src/views/IncorporationAgreement.vue

```diff
@@ -200,6 +200,11 @@ import { AgreementType } from '@/components/IncorporationAgreement'
 export default class IncorporationAgreement extends Vue {
   private helpToggle: boolean = false
   private readMoreFlag: boolean = false
+
+  private get documentURL ():string {
+    return sessionStorage.getItem('BASE_URL') +
+    'files/benefit_company__corporation_agreement.pdf'
```

Comment:
Are you sure this will work?
Sample Base URL: https://dev.bcregistry.ca/cooperatives/CP0000841/

Reply:
The base url that I see in my session is https://business-create-dev.pathfinder.gov.bc.ca/businesses/create/

Isnt that right?

Reply:
I think the one that you mentioned is for filings UI right?

Reply:
Yes, sorry, wrong project :)